mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
+	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
+	METHOD add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 value
```

Comment:
Hmm, seems mappingpoet generates java code for these bridge methods and causes javadoc errors. I will fix that later